independent-projects/bootstrap/pom.xml

```diff
@@ -42,13 +42,13 @@
         <eclipse-minimal-json.version>0.9.5</eclipse-minimal-json.version>
         <jboss-logging.version>3.6.0.Final</jboss-logging.version>
         <junit.jupiter.version>5.11.0</junit.jupiter.version>
-        <maven-core.version>3.9.8</maven-core.version><!-- Keep in sync with sisu.version -->
+        <maven-core.version>3.9.9</maven-core.version><!-- Keep in sync with sisu.version -->
         <sisu.version>0.9.0.M3</sisu.version><!-- Keep in sync with maven-core.version -->
-        <maven-plugin-annotations.version>3.13.0</maven-plugin-annotations.version>
+        <maven-plugin-annotations.version>3.13.1</maven-plugin-annotations.version>
         <maven-resolver.version>1.9.22</maven-resolver.version>
         <maven-shared-utils.version>3.4.2</maven-shared-utils.version> <!-- Keep in sync with maven-core.version (force this version on maven-invoker) -->
         <maven-wagon.version>3.5.3</maven-wagon.version>
-        <httpcore.version>4.4.16</httpcore.version><!-- Keep in sync with maven-wagon.version (wagon-http 3.4.3 brings in 4.4.13 and 4.4.14) -->
+        <httpcore.version>4.4.16</httpcore.version><!-- Keep in sync with maven-wagon.version (wagon-http 3.5.3 brings in httpclient 4.5.14 which brings httpcore 4.4.16) -->
         <httpclient.version>4.5.14</httpclient.version>
         <commons-logging-jboss-logging.version>1.0.0.Final</commons-logging-jboss-logging.version>
         <jackson-bom.version>2.15.2</jackson-bom.version>
```

Comment:
unrelated to this, but should this match the rest of quarkus i.e. bump to 2.17.2?

Reply:
Good catch. Actually, it should be dropped as it's not used (and thus why Dependabot doesn't update it)